api/external/applications/applications.proto

```diff
@@ -469,27 +531,77 @@ message ServicesStatsRes {
 
 // Periodic job configuration.
 message PeriodicJobConfig {
-  // The job status? `false` is disabled, `true` is enabled.
+  // Enable/disable the job. `false` is disabled, `true` is enabled.
   bool running = 1;
   // The `threshold` setting used by periodic jobs for evaluating services.
   // Threshold is a string that follows Elasticsearch's date math expressions. For more information, see the simpledatemath package under `lib/`.
   string threshold = 2;
+  // A recurrence rule that determines how often, at what interval, and when to
+  // initially start a scheduled job.
+  //
+  // If the field is omitted from the request or is set to an emtpy string, no
+  // change will be made to the current value. Otherwise, the value should match
+  // the  [recurrence rule format defined in section 4.3.10 of RFC 2445](https://www.ietf.org/rfc/rfc2445.txt).
+  //
+  // It is not recommended to change this value from the default setting of a
+  // 60 second interval. This configuration option is provided only for
+  // consistency with other data lifecyle APIs.
+	string recurrence = 3;
+  PeriodicJobInfo job_info = 4;
 }
 
-// Configuration for the mandatory periodic job.
-// Required and cannot be disabled.
+// Configuration for a periodic job. Initially Jobs using this message type
+// were designed such that they could not be disabled, but that has been
+// changed to make the various data lifecycle APIs consistent with each other.
+// Thus, there is a `running` field which will disable the job if set to false.
+// That is not recommended.
 message PeriodicMandatoryJobConfig {
   // The `threshold` setting used by periodic jobs for evaluating services.
   // Threshold is a string that follows Elasticsearch's date math expressions. For more information, see the simpledatemath package under `lib/`.
   string threshold = 1;
+  // Enable/disable the job. `false` is disabled, `true` is enabled. It is not
+  // recommended to disable this job.
+  google.protobuf.BoolValue running = 2;
```

Comment:
I needed to use a wrapper here to preserve API compatibility. Existing users might have code/docs/scripts to send a JSON payload like

```json
'{"threshold": "15m" }'
```
If we used a plain boolean for the new element, it would be `false` whenever the user didn't include it in the JSON payload. Thus, the `running` setting would be false and disable the job. We _could_ have flipped the setting to be "disabled" (so a `false` value would make the job enabled), but then it would be different than the other job which would be ugly. 

Using a wrapper lets us distinguish between "not passed" and "false" and has the same JSON representation so it was the best option in the end.